src/UserGuide/V1.2.x/User-Manual/Tiered-Storage_timecho.md

```diff
@@ -19,6 +19,78 @@
+| object_storage_access_secret             |                          | Authentication information stored in the cloud: secret                                    | AWS S3 的 credential secret；If remote storage is not used, no configuration required    |
+| remote_tsfile_cache_dirs                 | data/datanode/data/cache | Cache directory stored locally in the cloud                                     | If remote storage is not used, no configuration required                                 |
+| remote_tsfile_cache_page_size_in_kb      | 20480                    |Block size of locally cached files stored in the cloud                               | If remote storage is not used, no configuration required                                 |
+| remote_tsfile_cache_max_disk_usage_in_mb | 51200                    | Maximum Disk Occupancy Size for Cloud Storage Local Cache                           | If remote storage is not used, no configuration required                                 |
+
+## local tiered storag configuration example
+
+The following is an example of a local two-level storage configuration.
+
+```JavaScript
+//Required configuration items
+dn_data_dirs=/data1/data;/data2/data,/data3/data;
+default_ttl_in_ms=86400000;-1
+dn_default_space_move_thresholds=0.2;0.1
+```
+
+In this example, two levels of storage are configured, specifically:
+
+| **tier** | **data path**                           | **data range**    | **threshold for minimum remaining disk space** |
+| -------- | -------------------------------------- | --------------- | ------------------------ |
+| tier 1   | path 1：/data1/data                    | data for last 1 day | 20%                      |
+| tier 2   | path 2：/data2/data path 2：/data3/data | data from 1 day ago  | 10%                      |
+
+## remote tiered storag configuration example
+
+The following takes three-level storage as an example:
+
+```JavaScript
+//Required configuration items
+dn_data_dirs=/data1/data;/data2/data,/data3/data;OBJECT_STORAGE
+default_ttl_in_ms=86400000;864000000;-1
+dn_default_space_move_thresholds=0.2;0.15;0.1
+object_storage_name=AWS_S3
+object_storage_bucket=iotdb
+object_storage_endpoiont=<your_endpoint>
+object_storage_access_key=<your_access_key>
+object_storage_access_secret=<your_access_secret>
+
+// Optional configuration items
+remote_tsfile_cache_dirs=data/datanode/data/cache
+remote_tsfile_cache_page_size_in_kb=20971520
+remote_tsfile_cache_max_disk_usage_in_mb=53687091200
+```
+
+In this example, a total of three levels of storage are configured, specifically:
+
+| **tier** | **data path**                           | **data range**                 | **threshold for minimum remaining disk space** |
+| -------- | -------------------------------------- | ---------------------------- | ------------------------ |
+| tier一   | path 1：/data1/data                    | data for last 1 day              | 20%                      |
```

Comment:
| tier   | path 1：/data1/data                    | data for last 1 day              | 20%                      |